src/frontend/src/components/navbar.ts

```diff
@@ -3,5 +3,5 @@ import { aboutLink } from "../components/aboutLink";
 import { faqLink } from "../components/faqLink";
 
 export const navbar = html`<div id="navbar">
-  ${aboutLink} &middot; ${faqLink}
+  <div class="nav-links">${aboutLink} &middot; ${faqLink}</div>
```

Comment:
can't `<div class="nav-links">` be spared?

Reply:
It's used to text-align: center